apps/console/src/app.tsx

```diff
@@ -61,6 +61,13 @@ import {
 } from "./features/core/models";
 import { AppState } from "./features/core/store";
 
+declare global {
```

Comment:
Shall we move this to the dedicated `types` folder.
https://github.com/wso2/identity-apps/blob/master/apps/console/src/types/global.d.ts